
# HG changeset patch
# User Vladimir Homutov <vl@nginx.com>
# Date 1469550852 -10800
# Node ID d43ee392e825186545d81e683b88cc58ef8479bc
# Parent  0b4249f975eba48c17b69c5e225fd12b470c0184
Stream: fixed build without stream_ssl_module (ticket #1032).

diff -r 0b4249f975eb -r d43ee392e825 src/stream/ngx_stream_proxy_module.c
--- a/src/stream/ngx_stream_proxy_module.c	Tue Jul 26 19:07:18 2016 +0300
+++ b/src/stream/ngx_stream_proxy_module.c	Tue Jul 26 19:34:12 2016 +0300
@@ -538,7 +538,7 @@
         return;
     }
 
-#if (NGX_HTTP_SSL)
+#if (NGX_STREAM_SSL)
     u->ssl_name = uscf->host;
 #endif
 
@@ -1157,7 +1157,7 @@
     u = s->upstream;
     ur = u->resolved;
 
-    ngx_log_debug0(NGX_LOG_DEBUG_HTTP, s->connection->log, 0,
+    ngx_log_debug0(NGX_LOG_DEBUG_STREAM, s->connection->log, 0,
                    "stream upstream resolve");
 
     if (ctx->state) {

